src/providers/authProvider/index.ts

```diff
@@ -18,7 +18,7 @@ import axios, { isAxiosError } from 'axios'
 import bcrypt from 'bcryptjs'
 
 export const getUser = (): _Users | undefined => {
-  const encryptedUser = getCookie(constants.TOKEN_KEY)
+  const encryptedUser = getCookie('accessToken')
```

Comment:
@TahaKhanAbdalli - we use this string here and at line 38.  Could we use a constant instead, please?